tools/snvphyl-tools/vcf2snvalignment/test-data/expected_stats.csv

```diff
@@ -1,3 +1,3 @@
-#Reference	total length	total invalid pos	total valid pos	total core	Percentage in core
-ref	4	N/A	4	4	100.00
-all	4	N/A	4	4	100.00
+#Reference name	Total length	Total invalid and excluded positions	Total valid and included positions	Total valid and included positions in core genome	Percentage of valid and included positions in core genome	Percentage of all positions that are valid, included, and part of the core genome
```

Comment:
Thanks for fixing up these tests files.

Reply:
Reduce total size for test for consolidate to 1.1MB.

Reply:
Thanks.